dex/src/main/scala/com/wavesplatform/dex/settings/MatcherSettings.scala

```diff
@@ -62,7 +62,8 @@ case class MatcherSettings(
   webSockets: WebSocketSettings,
   addressActor: AddressActor.Settings,
   orderEventsCoordinatorActor: OrderEventsCoordinatorActor.Settings,
-  comparisonTool: ComparisonTool.Settings
+  comparisonTool: ComparisonTool.Settings,
+  uncheckingConfigs: Option[String]
```

Comment:
`List[String]`, also it has to be wrapped in `cli: CliSettings`

Reply:
Done